src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx

```diff
@@ -17,6 +20,8 @@ const UserInfo = () => {
   const userName = useAppSelector(userNameSelector);
   const userPasswordSelector = selectUserPasswordByIndex(index);
   const userPassword = useAppSelector(userPasswordSelector);
+  const userSshKeySelector = selectUserSshKeyByIndex(0);
```

Comment:
Looking at the other selectors, shouldn't this also be like this?
```suggestion
  const userSshKeySelector = selectUserSshKeyByIndex(index);
```